unity-renderer/Assets/DCLPlugins/EmoteAnimations/EmoteAnimationsTracker.cs

```diff
@@ -106,10 +106,11 @@ private async UniTaskVoid LoadEmote(string bodyShapeId, string emoteId, Cancella
 
             try
             {
+                loaders.Add((bodyShapeId, emoteId), null);
                 var emote = await(emotesCatalogService.RequestEmoteAsync(emoteId, ct));
 
                 IEmoteAnimationLoader animationLoader = emoteAnimationLoaderFactory.Get();
-                loaders.Add((bodyShapeId, emoteId), animationLoader);
+                loaders[(bodyShapeId, emoteId)] = animationLoader;
```

Comment:
whats going to happen to the previous animation loader if overriden? should be disposed?